ci/ci/ci.py

```diff
@@ -108,10 +108,12 @@
     if pr.batch:
         if hasattr(pr.batch, 'id'):
             status = await pr.batch.status()
+            jobs = await collect_agen(await pr.batch.jobs())
```

Comment:
I don't think you want the await inside the parentheses. Either that or it should be below `await collect_agen(b.jobs())`